src/presentation/profile/EditProfile.js

```diff
@@ -173,9 +207,10 @@ class EditProfile extends React.Component {
       })
     const userLinks = get(userProfile, 'links', [])
     const userCredits = get(userProfile, 'credits', [])
-    const profileImageUrl = get(profile, 'photoURL', '')
+    const photoURL = get(profile, 'photoURL', '')
     const name = `${get(profile, 'firstName', '')} ${get(profile, 'lastName', '')}`
-    const email = get(auth, 'email', '')
+    const youtubeVideo = get(profile, 'youtubeVideo', '')
+    const vimeoVideo = get(profile, 'vimeoVideo', '')
```

Comment:
These are incorrect. react-redux-firebase names things terribly. So account is actually profile here ( they are dumb). If you want to access the actual profile its in `firebase.data.userProfiles.uid`